config/initializers/govuk_notify.rb

```diff
@@ -1,2 +1,14 @@
 ActionMailer::Base.add_delivery_method :govuk_notify, GovukNotifyRails::Delivery,
-                                       api_key: ENV.fetch('GOVUK_NOTIFY_API_KEY', nil)
+                                       api_key: ENV.fetch("GOVUK_NOTIFY_API_KEY", nil)
+
+class MailDeliveryJobWrapper < ActionMailer::MailDeliveryJob
+  def serialize
+    super.tap do |args|
+      args["arguments"][3] = Array(args["arguments"][3])
```

Comment:
This feels a little bit nervous-making - are we sure that `args["arguments"][3]` is always going to be the `to` field? The fact that we now have this string-to-array issue covered nicely in `EmailToProviderMailer` makes me feel like this wrapper is possibly more of a liability than a win.

Reply:
yeah, basing it on an index feels brittle - might have string-to-arraying higher up the chain?

Reply:
The test always passes, so I'm confident enough to say yes it is. It is a bit scary though